db/migrate/20160226221827_rename_solutions_to_collaborator_solutions.rb

```diff
@@ -0,0 +1,23 @@
+class RenameSolutionsToCollaboratorSolutions < ActiveRecord::Migration
+  def up
+    # Transfer any solution Attachments to their Exercises
+    Attachment.where(parent_type: 'Solution').preload(:parent).find_each do |attachment|
+      attachment.update_attribute(:parent, attachment.parent.question.exercise)
+    end
```

Comment:
I *think* they already go to the Exercise according to https://github.com/openstax/exercises/blob/master/app/routines/exercises/import/row_importer.rb#L44, that line was mostly to be safe...